utils/check_repo.py

```diff
@@ -30,6 +30,9 @@
 # Being in this list is an exception and should **not** be the rule.
 IGNORE_NON_TESTED = [
     # models to ignore for not tested
+    "Speech2TextTransformerEncoder",  # Building part of bigger (tested) model.
+    "Speech2TextTransformerDecoder",  # Building part of bigger (tested) model.
+    "Speech2TextTransformerDecoderWrapper",  # Building part of bigger (tested) model.
```

Comment:
Think those classes: `Speech2TextTransformerDecoderWrapper` can be deleted for now